app/src/main/java/com/vitorpamplona/amethyst/ui/screen/loggedIn/ProfileScreen.kt

```diff
@@ -317,7 +319,9 @@ private fun ProfileHeader(
                         MaterialTheme.colors.background,
                         CircleShape), 
                     onClick =  {
-                        navController.navigate("User/${it.pubkeyHex}")
+                        if (baseUser.profilePicture() != null) {
+                            zoomImageDialogOpen = true
+                        }
```

Comment:
Open the dialog on click (previously it navigated to the page we were already on, so no loss of functionality here)